assets/l10n/app_en.arb

```diff
@@ -43,8 +43,8 @@
   "@permissionsDeniedReadExternalStorage": {
     "description": "Message for dialog asking the user to grant permissions for external storage read access."
   },
-  "actionSheetOptionCopy": "Copy message text",
-  "@actionSheetOptionCopy":  {
+  "actionSheetOptionCopyMessageText": "Copy message text",
+  "@actionSheetOptionCopyMessageText":  {
```

Comment:
Cool. I think I'd like to rename `successMessageCopied` to `successMessageTextCopied`, too.

I think it would also be good to make that UI string more parallel to this existing "Copy message text" button. Instead of "Message Copied" (which has the wrong capitalization anyway), it could say "Message text copied".

Changing the user-facing text won't be NFC, so if it goes in this commit, the commit should be unmarked `[nfc]`. I don't think I have a preference if that change goes here or in a different commit.